Makefile

```diff
@@ -20,7 +20,7 @@ test_integration:
 	- ${DOCKER_COMPOSE_TEST} down
 	${DOCKER_COMPOSE_TEST} up -d
 	./.scripts/docker/wait-healthy.sh test_postgres 20
-	./.scripts/docker/wait-healthy.sh test_rabbitmq 20
+	./.scripts/docker/wait-healthy.sh test_rabbitmq 60
```

Comment:
spinning up rabbit takes < 10 seconds on my machine - do we know what is going on?

Reply:
https://github.com/libero/scripts/blob/master/docker/wait-healthy.sh prints one line per second, so [outputs such as](https://travis-ci.com/libero/continuum-adaptor/builds/145218843):
```
test_rabbitmq is in state: starting
test_rabbitmq is in state: starting
test_rabbitmq is in state: starting
test_rabbitmq is in state: starting
test_rabbitmq is in state: healthy
``` 
point to a few seconds of startup time. Anyone has a link to a build that failed because of the 20-second timeout?

Reply:
Takes over 40 seconds on my machine...

Reply:
Let's check that in person, could be anything from some volume state to other containers trashing?